iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m

```diff
@@ -107,10 +106,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
-#define ONESIGNAL_APP_ID_KEY_FOR_TESTING @"77e32082-ea27-42e3-a898-c72e141824ef"
+#define ONESIGNAL_APP_ID_KEY_FOR_TESTING @"ONESIGNAL_APP_ID_KEY_FOR_TESTING"
 
 + (NSString*)getOneSignalAppId {
-    NSString* newAppId = @"77e32082-ea27-42e3-a898-c72e141824ef";
+    NSString* newAppId = @"0ba9731b-33bd-43f4-8b59-61172e27447d";
```

Comment:
This can stay as the default (`0ba9731b-33bd-43f4-8b59-61172e27447d`) as well, but the point is that we can easily chan get `appId` and rerun without having to worry about the app using the cached one but showing in the UI the new one.